cropharvest/eo/eo.py

```diff
@@ -100,7 +100,6 @@ class EarthEngineExporter:
 
     def __init__(
         self,
-        labels: Optional[geopandas.GeoDataFrame] = None,
```

Comment:
I think labels are irrelevant when using the EarthEngineExporter for exporting a region or a bounding box, hence I've moved this argument into `export_for_labels`